images/win/scripts/Installers/Validate-ToolCache.ps1

```diff
@@ -22,36 +37,53 @@ function ToolcacheTest {
     if (Test-Path "$env:AGENT_TOOLSDIRECTORY\$SoftwareName")
     {
         $description = ""
-        [array]$versions = GetChildFolders -Path "$env:AGENT_TOOLSDIRECTORY\$SoftwareName"
-        if ($versions.count -gt 0){
-            foreach ($version in $versions)
+        [array]$instaledVersions = GetChildFolders -Path "$env:AGENT_TOOLSDIRECTORY\$SoftwareName"
```

Comment:
typo in `installed` - should be double 'L'

Reply:
Fixed.